packages/react/src/auto/polaris/PolarisAutoForm.tsx

```diff
@@ -32,8 +32,12 @@ export const PolarisAutoForm = <
   const { action, findBy } = props as AutoFormProps<GivenOptions, SchemaT, ActionFunc> &
     Omit<Partial<FormProps>, "action"> & { findBy: any };
 
+  if (!action) {
+    throw new Error("A valid Gadget action is required for AutoForm components");
+  }
+
   // Component key to force re-render when the action or findBy changes
-  const componentKey = `${action.modelApiIdentifier}.${action.operationName}.${findBy}`;
+  const componentKey = `${action.modelApiIdentifier ?? ""}.${action.operationName}.${findBy}`;
```

Comment:
Why do we need this `??` fallback?

Reply:
Global actions don't have `modelApiIdentifier`, and they previously had `undefined` in the key string